CMakeLists.txt

```diff
@@ -272,11 +272,15 @@ elseif(MSVC)
                     #/D_HAS_ITERATOR_DEBUGGING=0
     )
 
+    # thomthom : When enabling static CRT there are linker errors in Release
+    # referring to libcpmt (Note not libcmt). Not sure why, but removing this
+    # exclude allowed it to build, and there where no linker error in VS2013/15
+    # when building with minimal build.
     # Turn off a duplicate LIBCMT linker warning
-    set(CMAKE_EXE_LINKER_FLAGS
-        "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:libcmt.lib")
-    set(CMAKE_SHARED_LINKER_FLAGS
-        "${CMAKE_SHARED_LINKER_FLAGS} /NODEFAULTLIB:libcmt.lib")
+    #set(CMAKE_EXE_LINKER_FLAGS
```

Comment:
This is my biggest concern. I don't understand why I was getting `libcpmt.lib` linker errors only in Release. And why the `cp` version and not the `c` version which is actually excluded.

Linker error log: https://gist.github.com/thomthom/5f9a23263c6a05ec5c5d

I did not observe any linker warnings by removing this. But maybe that's because I tried with a minimal config.

Anyone got any ideas to what is going on here?